spec/features/items/update_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+RSpec.describe "Items Show Page" do
+  before(:each) do
+    @merchant = create(:merchant)
+    @item = @merchant.items.create!(name: "Candle", description: "Pine Scented", unit_price: 1000)
+    @item2 = @merchant.items.create!(name: "Soap", description: "Fresh Spring", unit_price: 1500)
+    @item3 = @merchant.items.create!(name: "Coaster", description: "Mahogany", unit_price: 500)
+  end
+
+# User Story 8 - Merchant Item Update
+# As a merchant,
+# When I visit the merchant show page of an item (/merchants/:merchant_id/items/:item_id)
+# I see a link to update the item information.
+# When I click the link
+# Then I am taken to a page to edit this item
+# And I see a form filled in with the existing item attribute information
```

Comment:
do you have an assertion testing that the existing attribute info is there in the form before you fill in the form?